flow/client/flow_client.go

```diff
@@ -158,13 +158,16 @@ retry:
 	return nil
 }
 
-// SendFlows sends flows to the server
-func (c *FlowClient) SendFlows(flowArray *flow.FlowArray) {
-	// TODO(safchain) in case of websocket there is no size limitation we should send
-	// bulk directly
-	for _, flow := range flowArray.Flows {
-		if err := c.SendFlow(flow); err != nil {
-			logging.GetLogger().Errorf("Unable to send flow: %s", err)
+// SendFlows implements the flow Sender interface
+func (c *FlowClient) SendFlows(flows []*flow.Flow) {
+	if len(flows) > 0 {
```

Comment:
since we are using ```range ```, do we need this check?

Reply:
indeed stupid :)